src/app/components/search/search.component.ts

```diff
@@ -15,6 +15,6 @@ export class SearchComponent {
   */
   filterCategories = new Map([
     ['Technologies', ['IMS', 'MxIF', 'AF', 'PAS', 'IHC']],
-    ['TMCs', ['TMC-Vanderbilt', 'TMC-UCSD', 'TMC-Stanford', 'TMC-Florida', 'TMC-CalTech']]
+    ['TMCs', ['Vanderbilt', 'UCSD', 'Stanford', 'Florida', 'CalTech']]
```

Comment:
This will probably not work as it will affect the state. Have you checked that the filtering functionality still works as intended?

Reply:
oh do you mean it wont filter?

Reply:
Yes, that is very likely

Reply:
Rather than removing the prefixes from the data it should be done during the display

Reply:
how about doing it in the template like this 
{{ filter.indexOf('TMC-') !== -1 ? (filter | slice:4:filter.length) : filter }}

Reply:
- Make a function for it and call it in the template. We might want to add/change this later.
- Also your current solution would remove the first 4 chars whenever the string TMC- is in the string -> foobarTMC- becomes arTMC-